src/main/java/uk/gov/hmcts/darts/audio/mapper/GetAudioRequestResponseMapper.java

```diff
@@ -25,9 +25,9 @@ public GetAudioRequestResponseV1 mapToAudioRequestSummary(EnhancedMediaRequestIn
         response.setMediaRequestStatus(MediaRequestStatus.fromValue(enhancedMediaRequestInfo.getMediaRequestStatus().toString()));
         response.setLastAccessedTs(transformedMedia.getLastAccessed());
         response.setOutputFilename(transformedMedia.getOutputFilename());
-        String outputFilename = transformedMedia.getOutputFormat();
+        uk.gov.hmcts.darts.audio.enums.AudioRequestOutputFormat outputFilename = transformedMedia.getOutputFormat();
         if (outputFilename != null) {
-            response.setOutputFormat(AudioRequestOutputFormat.fromValue(outputFilename));
+            response.setOutputFormat(AudioRequestOutputFormat.fromValue(outputFilename.getExtension()));
```

Comment:
Wouldn't we just need to use `toString` on the AudioRequestOutputFormat enum, so no need for all this `getExtension` stuff?

Reply:
variable required to return value when enum has string values